Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/infinitybase/InfinityConfig.kt

```diff
@@ -0,0 +1,22 @@
+package org.dolphinemu.dolphinemu.features.infinitybase
+
+object InfinityConfig {
+    var LIST_FIGURES: Map<Long, String> = getFigureMap()
+    var REVERSE_LIST_FIGURES: Map<String, Long> = getInverseFigureMap()
+
+    private external fun getFigureMap(): Map<Long, String>
+    private external fun getInverseFigureMap(): Map<String, Long>
+
+    @JvmStatic
+    external fun removeFigure(position: Int)
+
+    @JvmStatic
+    external fun loadFigure(position: Int, fileName: String?): String
```

Comment:
```suggestion
    external fun loadFigure(position: Int, fileName: String?): String?
```

Reply:
Also, I don't think the implementation of `loadFigure` actually works correctly when `fileName` is `null`. In that case, the `fileName` parameter should use `String` instead of `String?`.